src/components/button/primaryButton.jsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by jonlazarini on 14/02/17.
+ */
+import React from 'react';
+import Button from './index.jsx';
+import styled from 'styled-components';
+
+
+// Style wrapper for default buttons - Composition
+const PrimaryWrapper = (Comp) => styled(Comp)`
```

Comment:
just noticed the linting was not running in the background and there are a few more other errors which I will fix!